sdk/provisioning/Azure.Provisioning.ApplicationInsights/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 0.2.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 0.2.0 (2024-05-14)
```

Comment:
Is there a missing .csproj version change that should go along with this?

Reply:
Actually, it was the other way around - the changelog entry should not be here as we don't need to re-ship this library.